qa/L0_grpc_state_cleanup/cleanup_test.py

```diff
@@ -67,9 +67,10 @@ class CleanUpTest(tu.TestResultCollector):
     def setUp(self):
         self.decoupled_model_name_ = "repeat_int32"
         self.identity_model_name_ = "custom_zero_1_float32"
+        self.repeat_non_decoupled_model_name = "repeat_int32_non_decoupled"
 
     def _prepare_inputs_and_outputs(self, kind):
-        if kind == "decoupled_streaming":
+        if kind == "decoupled_streaming" or kind == "non_decoupled_streaming":
```

Comment:
nit: if kind in ("...", "...")

Reply:
I'm ok either way, but use the
```python
if kind in ("kind_1", "kind_2", ...):
...
```
form because the code is shorter.

Reply:
[Use in tuple instead of or](https://github.com/triton-inference-server/server/pull/7265/commits/178b65b955dc48222d701e08d6764847be0ebafc)